### PR TITLE
Update getting-started-runtime-configuration.md

### PR DESCRIPTION
**Pull request by jenworthington**
A lot of changes, with likely more to go. Take a look and let me know if you have comments, suggestions, etc., and I will do another edit.

### CODE REVIEW

**Comment by mstopa-splunk** [comment]
@jenworthington please pull my latest changes, it's been quite a lot of them exceeding this single document, because much of it's content was duplicating other articles. Anyway, ready for your final pass

**Comment by jenworthington** [comment]
Let's walk through how I pull your changes in the meeting tomorrow?

**Comment by mstopa-splunk** [comment]
@jenworthington ready for the final pass

File: docs/gettingstarted/quickstart_guide.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Quickstart Guide
+This guide will enable you to quickly implement basic changes to your Splunk instance and set up a simple SC4S installation. It's a great starting point for working with SC4S and establishing a minimal operational solution. The same steps are thoroughly described in the [Splunk Setup](getting-started-splunk-setup.md) and [Runtime configuration](getting-started-runtime-configuration.md) sections.
```
[the remark this answers is not in the document]
Comment (mstopa-splunk) [reply docs/gettingstarted/quickstart_guide.md:2]: @jenworthington please check this

File: docs/gettingstarted/getting-started-runtime-configuration.md [hunk 2]
```diff
@@ -1,56 +1,63 @@
...
-
-* Linux host with Docker (CE 19.x or greater) or Podman enabled, depending on runtime choice (below).
-* A network load balancer (NLB) configured for round-robin. Note: Special consideration may be required when more advanced products are used.
-The optimal configuration of the load balancer will round-robin each http POST request (not each connection).
-* The host linux OS receive buffer size should be tuned to match the sc4s default to avoid dropping events (packets) at the network level.
-The default receive buffer for sc4s is set to 16 MB for UDP traffic, which should be OK for most environments.  To set the host OS kernel to
-match this, edit `/etc/sysctl.conf` using the following whole-byte values corresponding to 16 MB:
+## Step 1: Configure your OS to work with SC4S  
+### Tune your receiver buffer
+The host Linux OS receiver buffer size must be tuned to match the SC4S default. This helps to avoid event dropping at the network level.
```
[the remark this answers is not in the document]
Comment (mstopa-splunk) [reply docs/gettingstarted/getting-started-runtime-configuration.md]: @jenworthington please check this

File: docs/gettingstarted/getting-started-runtime-configuration.md [hunk 3]
```diff
@@ -1,56 +1,63 @@
...
+The default receiver buffer for SC4S is 16 MB for UDP traffic, which should be acceptable for most environments. To set the host OS kernel to
```
[the remark this answers is not in the document]
Comment (mstopa-splunk) [reply docs/gettingstarted/getting-started-runtime-configuration.md:7]: @jenworthington please check

File: docs/gettingstarted/getting-started-runtime-configuration.md [hunk 4]
```diff
@@ -1,56 +1,63 @@
...
-unpredictable and difficult to detect. See this helpful discussion in the syslog-ng
-[Professional Edition](https://www.syslog-ng.com/technical-documents/doc/syslog-ng-premium-edition/7.0.10/collecting-log-messages-from-udp-sources)
-documentation regarding tuning syslog-ng in particular (via the [SC4S_SOURCE_*_SO_RCVBUFF](../configuration.md#syslog-source-configuration)
-environment variable in sc4s) as well as overall host kernel tuning.  The default values for receive kernel buffers in most distros is 2 MB,
-which has proven inadequate for many.
 
-## IPv4 Forwarding
+3. To verify that the kernel does not drop packets, periodically monitor the buffer using the command
+`netstat -su | grep "receive errors"`. Failure to tune the kernel for high-volume traffic results in message loss, which can be 
+unpredictable and difficult to detect. The default values for receiver kernel buffers in most distributions is 2 MB,
```
[the remark this answers is not in the document]
Comment (mstopa-splunk) [reply docs/gettingstarted/getting-started-runtime-configuration.md:23]: @jenworthington please check

**Pull request merged**